### DISCUSSION TITLE
Disable sky box rendering.

### DISCUSSION THREAD

**Discussion by Hipshot-zz** [Garux/netradiant-custom#173] [Q&A] [answered]
I can't find an option to get rid of the skybox rendering in the editor, is this possible at all?

I just want to point out that I want to see the wall as older radiant, because if I disable the "sky view" in the filter options I don't see the brush or anything.

**Comment by Garux** [comment]
There is no option to disable skybox rendering, but there is this option, helping to recognize brushes:
![image](https://github.com/Garux/netradiant-custom/assets/1734117/9464e027-525c-4384-b8a0-b37b42b37e46)

**Comment by Hipshot-zz** [comment]
Ah, very annoying, I can see why some want this feature though. For me it's in the way, it creates an illusion I don't need in the editor.

Thanks for the quick answer though, the editor has a lot of quality functions.

**Comment by Hipshot-zz** [comment]
Btw, if this happens to be looked at in the future, an idea here is that if a sky shader uses the "qer_editorimage" param, it would not render the sky, but the editor image instead and act like a regular brush. Might be a thing.

**Comment by Garux** [comment, marked as answer]
This would mean no wysiwyg for most of existing shaders :b
Real solution would be to implement the option, however it appears nuff usable with wireframe enabled.
If rendering is unwanted that badly, it may be worked around, e.g. 'cleanly' via `q3map_deprecateshader` or 'dirtily' by removing `_dn` image.

**Reply by Hipshot-zz** [reply]
Ah haha, ofc, I meant, in combination with a surfaceparm sky shader =) But yea, it could just be a real option. And I did solve it localy by just uncommenting the skyshader texture, but it's a bit messy to do all the time.
But that remove _dn worked better, thanks for the tip.
